src/internal/Component.js

```diff
@@ -0,0 +1,158 @@
+import { add_render_callback, flush, intro, schedule_update } from './scheduler.js';
+import { current_component, set_current_component } from './lifecycle.js'
+import { is_function, run, run_all, noop } from './utils.js';
+import { blankObject } from './utils.js';
+import { children } from './dom.js';
+
+export function bind(component, name, callback) {
+	component.$$.bound[name] = callback;
+	callback(component.$$.get()[name]);
+}
+
+export function mount_component({ $$: { fragment }}, target, anchor, hydrate) {
```

Comment:
```suggest
 export function mount_component(component, target, anchor, hydrate) { 
```

Reply:
thanks — yeah, i have this locally along with a few other things. haven't pushed them yet in case i need to backtrack (trying to get CEs working)

Reply:
Okay, cool. Just fyi in case it messes you up - about half an hour ago I pushed a fix to the root `index.js` which you should of course feel free to just `push -f` over